cpp/cmake/thirdparty/get_kvikio.cmake

```diff
@@ -25,10 +25,10 @@ function(find_and_configure_kvikio VERSION)
     OPTIONS "KvikIO_BUILD_EXAMPLES OFF"
   )
 
-  if(KvikIO_BINARY_DIR)
-    include("${rapids-cmake-dir}/export/find_package_root.cmake")
-    rapids_export_find_package_root(BUILD KvikIO "${KvikIO_BINARY_DIR}" cudf-exports)
-  endif()
+  include("${rapids-cmake-dir}/export/find_package_root.cmake")
```

Comment:
The CONDITION is new to me. But this change means that we include a file unconditionally where we used to conditionally conclude it. Does this need a condition?

Reply:
The include was only conditional so that we never brought in functions we didn't call. 

We won't have any problems always bringing in the file.